### ISSUE TITLE
Please provide an option to disable shortcuts (fixed)

### ISSUE THREAD

**Issue by xiaoouwang** [terrylinooo/githuber-md#277]
Hello dear author,

Tks for the plugin, it's fantastic :)

It has been mentioned some times but I'd like to emphasize how inconvenient it is for some people to not be able to turn off shortcuts.

In my case, ctrl+E is overridden by the emoji controller, which is kind of useless imo.

Keep up the good work!

Best,

**Comment by xiaoouwang** [comment]
For all people struggling with shortcuts, here is a quick fix (I tweaked some parameters, use it at ur own risk):

Use the plugin source file editor, go to `wp-githuber-md/assets/vendor/editor.md/editormd.min.js`, search the file using `ctrl-` and you will see all the shortcuts. Delete those that you don't want.

Then, go to `wp-githuber-md/src/Controllers/Markdown.php`, and search for `wp_enqueue_script( 'editormd', $this->githuber_plugin_url . 'assets/vendor/editor.md/editormd.min.js', array( 'jquery' ), $this->editormd_varsion, true );`

Delete `, array( 'jquery' ), $this->editormd_varsion, true )`

Voila, hope it helps :)

In short, it's not a problem of this plugin so it's a little tricky for the author to provide some config options I guess.
